src/integrationtests/java/com/aws/greengrass/integrationtests/certificateauthority/OfflineAuthenticationTest.java

```diff
@@ -194,10 +195,10 @@ void GIVEN_storedCertificates_WHEN_refreshEnabled_THEN_storedCertificatesRefresh
        String clientBPem = CertificateHelper.toPem(clientBCrt);
        iotAuthClientFake.activateCert(clientBPem);
        when(networkStateMock.getConnectionStateFromMqtt()).thenReturn(NetworkState.ConnectionState.NETWORK_UP);
-       String thingOne = "ThingOne";
-       iotAuthClientFake.attachCertificateToThing(thingOne, clientAPem);
-       String thingTwo = "ThingTwo";
-       iotAuthClientFake.attachCertificateToThing(thingTwo, clientBPem);
+       Supplier<String> thingOne =  () -> "ThingOne";
```

Comment:
This was so we could simulate failures by passing to the fake something that could explode